src/python/grpcio/grpc/experimental/aio/__init__.py

```diff
@@ -11,7 +11,11 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-"""gRPC's Asynchronous Python API."""
+"""gRPC's Asynchronous Python API.
+
+gRPC Async API objects may only be used on the thread on which they were
```

Comment:
Will this comment show up in our reference docs? That would be ideal.

Reply:
https://github.com/grpc/grpc/issues/21400

Reply:
It won't yet. But it should be.

Adding a TODO issue at https://github.com/grpc/grpc/issues/21400.